Cargo.toml

```diff
@@ -1,8 +1,12 @@
 [package]
 name = "vmm-sys-util"
 version = "0.1.0"
-authors = ["Jing Liu <jing2.liu@linux.intel.com>"]
-license = "Apache-2.0"
+authors = ["Intel Virtualization Team <vmm-maintainers@intel.com>"]
```

Comment:
Please state in the commit message that this is fixing https://github.com/rust-vmm/vmm-sys-util/issues/13

Reply:
Also for the next commit, specify that it fixes https://github.com/rust-vmm/vmm-sys-util/issues/2

Reply:
Comments are addressed. Thanks!